lifecycle/gunicorn.conf.py

```diff
@@ -30,10 +30,11 @@
 
 defuse_stdlib()
 
-try:
-    backend._enable_fips()
-except InternalError:
-    pass
+if CONFIG.get_bool("compliance.fips.enabled", False):
+    try:
```

Comment:
I think with it being an explicit option to enable, when the option is enabled and we can't active it, we should throw that error instead of just continuing